recovery/recovery.qdoc

```diff
@@ -203,7 +207,19 @@
   -# runs custom recovery (or built-in implementation of it) against merged results.
 
 
-  Built-in implementation of custom recovery for each key from merged results:\n
+  Built-in implementation of custom recovery for each key from merged results consists of two steps: server-send recovery with following rest-keys recovery.\n
+  If key could not be recovered via server-send, then it will be processed during rest-keys recovery.\n
+
+  Server-send recovery:
+  -# while there are non-empty buckets:
+    -# get next bucket - file that contains newest keys from a specific group
+    -# iterate over bucket keys:
+      -# check that key could be recovered via server-send, if so:
+        -# get destination groups to which key must be recovered and call server-send for it
```

Comment:
According to this, recovery calls server-send for each keys. Doesn't it combine keys into batches and call server-send for each batches, does it?